src/components/calcite-combobox/calcite-combobox.e2e.ts

```diff
@@ -327,4 +327,118 @@ describe("calcite-combobox", () => {
       expect(await item1.getProperty("selected")).toBe(true);
     });
   });
+
+  describe("single select", () => {
+    it("should allow selection of single item", async () => {
+      const page = await newE2EPage();
+      await page.setContent(
+        html`
+          <calcite-combobox selection-mode="single">
```

Comment:
selection-mode is an invalid prop as it's not defined anywhere on combobox.

Reply:
https://github.com/Esri/calcite-components/pull/1482/files#diff-3c5e42fd5fba0b231584840e4808faf46b8f2cdbda50c5b3bdd1b86dfa779378R77